### ISSUE TITLE
metrics

### ISSUE THREAD

**Issue by bysb** [lihaolin88/3D-Half-Gaussian-Splatting#5]
Excuse me, after training on the truck dataset (which I have trained many times), the final metrics are not as good as those in your paper. Have you made any modifications to your code？

**Comment by lihaolin88** [comment]
Hello, thank you for your interest! Performance may vary slightly depending on the machine or version of packages in the environment. To achieve better performance, you can try increasing the learning rate for normals in "half_gaussian_project/scene/gaussian_model" at line 176 ({'params': [self._normal], 'lr': 0.003, 'name': 'normal'}). You can find additional hints in Figure 6 of our paper.

**Comment by lihaolin88** [comment]
Hello, thank you so much for your report! We reviewed our recent code push on GitHub and identified the version used for fine-tuning the network. Specifically, in the file ./scene/gaussian_model.py at line 193, there is a "+15000" added.

Using this value during training from scratch causes the network to adopt a very small learning rate, leading to suboptimal results. To train from scratch, we should remove the "+15000" from line 193.

We have fixed this issue today, and the new version should now produce a score similar to what we reported in the paper.
